src/components/AssetManagementPanel/AssetManagementApplication/index.tsx

```diff
@@ -27,10 +28,34 @@ export const AssetManagementApplication = ({
   );
   const { send: sendSurrender, state: surrenderingState } = useContractFunctionHook(titleEscrow, "transferTo");
 
+  const { send: changeHolder, state: holderState } = useContractFunctionHook(titleEscrow, "changeHolder");
+
   const onSurrender = () => {
     sendSurrender(tokenRegistryAddress);
   };
 
+  const onTransfer = (nextHolder: string) => {
```

Comment:
This fn is redundant, can just put `changeHolder` whereever you put `onTransfer`

Reply:
ok understand, removed extra fn.